content/vendor/helm/helm-processing.md

```diff
@@ -1,23 +1,104 @@
 ---
 date: 2020-01-03
-linktitle: "Replicated Helm Processing"
-title: Replicated Helm Processing
+linktitle: "Helm Processing"
+title: Helm Processing
 weight: 20590
 ---
 
-The Replicated Helm installation is compatible with both Helm V2 and Helm V3 as it does not rely on Tiller to deploy the chart into the cluster. 
-Instead, KOTS treats a Helm Chart as the packaging spec rather than the deployment tool. 
+The Replicated Helm installation is compatible with both Helm V2 and Helm V3 as it does not rely on Tiller to deploy the chart into the cluster.  
```

Comment:
I felt like without the line breaks, the paragraphs were too thick and started to become a wall of text.  Including an empty line between the sentences disrupted the flow, so this seemed like a good happy medium.  For your plugin, is it possible to exclude .md files?  The preferred method for new lines is double-whitespace https://www.markdownguide.org/basic-syntax/#line-breaks

Reply:
with text wrapping, i've never felt like it reads poorly and consecutive lines without double line breaks render to single paragraphs, right? it's just personal preference though so i suggest leaving as is

Reply:
Double whitespace at the end of the line means "new line in this paragraph".  So these sentences are separated by a single line break on the page, like this:
![image](https://user-images.githubusercontent.com/83238463/145616389-97bcb591-e728-4b69-999f-347255802f03.png)

Helps break the wall of text feel of the paragraphs, I think

Reply:
I use Atom as my preferred editor and it automatically deletes trailing whitespaces when you commit a file.

Reply:
For the line breaks, if we're trying to split the paragraph for clarity and to give the reader space ....the general default is to create 2 paragraphs vs moving a new sentence to the next line.

Reply:
I can replace the double spaces with <br> as suggested by the markdown guide (https://www.markdownguide.org/basic-syntax/#line-breaks) if that's preferred.  Though if markdown officially supports trailing whitespace as a formatting function, you could look into excluding .md files from the auto-formatting in atom, like: 
![image](https://user-images.githubusercontent.com/83238463/145617625-c07dda7c-c007-4cc2-b470-1c094607ae5f.png)

Reply:
I've restructured the PR to eliminate all single line breaks.

Reply:
Would it make sense if we explain _why_ it doesn't rely on tiller? The real reason is because replicated is creating the resources not helm (via apply) afaik, right?